docs/zkapps/tutorials/03-deploying-to-a-network.mdx

```diff
@@ -105,27 +105,22 @@ Start by deleting the default files that come with the new project.
   $ rm src/Add.ts
   $ rm src/Add.test.ts
   $ rm src/interact.ts
+  $ rm src/index.ts
```

Comment:
Although this doesn't effect how the tutorial actually runs, the `index.ts` file does not need to be removed here.